### PR TITLE
Add persistant storage

### PR DESCRIPTION
**Pull request by limavfabio**
This PR adds persistent storage functionality for the application.

### CODE REVIEW

File: src/persist_data/persist_books.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+require 'json'
+
+module PersistBooks
+  SOURCE = './DB/books.json'.freeze
+
+  def self.read_from_file
+    deserialized_books = JSON.parse(File.read(SOURCE))
```
Comment (omarsalem7) [inline src/persist_data/persist_books.rb:9]: - [x] Kindly handle the case when the file is missing for (books,people, rentals), you can handle this by adding the following line
```ruby
 return [] unless File.exists?(SOURCE)
```

**Review by AmaduKamara — COMMENTED** [review]
# Status: Invalid Submission ❌ ❌ ❌

Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is using a PR that is not meant for this milestone.

- [ ] We are not allowed to use the same branch/PR for multiple milestones, but each implementation of the project requirements of one milestone should have its corresponding branch/PR.
- [ ] Currently, you created your PR on the preserve_data branch, but the milestone is, Activity name: OOP school library: unit tests, therefore, you should have created the PR on a test branch, which I noticed you're already having `add_unit_tests_with_rspec` branch.
- [ ] Kindly create your PR on a milestone for which your tasks are meant for all projects. 👍🏽

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
```

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @limavfabio, @Camilovelag 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- Correct git-flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/persist_data/persist_books.rb (hunk 1 above)
Comment (Camilovelag) [reply src/persist_data/persist_books.rb:9]: Done! 😄

**Review by BenMukebo — APPROVED** [review]
### Hi @TEAM, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



### To Highlight :1st_place_medal: 
- Gitflow has been used ✔️  
- No Linter errors :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

``` ## Authors ```
- [OPTIONAL] Update your Readme, This is a group project. Please let's include all the authors in the README file. This makes the README file professional and descriptive. 👨🏻‍🎨 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**